modules/apps/frontend-js/frontend-js-loader-modules-extender/src/main/java/com/liferay/frontend/js/loader/modules/extender/internal/servlet/JSResolveModulesServlet.java

```diff
@@ -76,6 +94,11 @@ protected void service(
 		printWriter.close();
 	}
 
+	protected void updateEtag() {
```

Comment:
I think this might have to be `updateETag`?

Reply:
:+1: